docs/css/style.css

```
@@ -146,6 +146,7 @@ a.direct-link {
   text-indent: -60px;
   transform: translateX(-100%);
   width: 30px;
+  min-height: 1.55em;
```

I think hiding the negatively indented text with an `overflow` setting is probably the correct way to go

```suggestion
  overflow: hidden;
```

---

@Munter that worked, was able to avoid making the change in .eleventy js (changing line 24 from `    permalinkSymbol: '#'` to `    permalinkSymbol: ' '`) as well. lmk if you still wanted that made